deps.java

```diff
@@ -1,14 +1,20 @@
 ///usr/bin/env jbang "$0" "$@" ; exit $?
 
+// - https://mvnrepository.com/artifact/dev.jbang/jash
```

Comment:
fine merging it ...but whats those links for ?

Reply:
I use them in the IDE to go and look at the latest versions of a dependency. Allows me to decide whether to select a minor update or major version update. They are clickable; very useful.